evcouplings/couplings/mapping.py

```diff
@@ -326,22 +362,9 @@ def segment_map_ecs(ecs, mapper):
     """
     ecs = deepcopy(ecs)
```

Comment:
can you please update this from my 2016 pandas knowledge to "ecs.copy()"? thanks ;)

Reply:
D: I've been using deepcopy on dataframes this whole time! Good to know